src/main/resources/reference.conf

```diff
@@ -998,14 +998,29 @@ resourceTypes = {
       "alter_policies" = {
         description = "Can alter policies"
       }
+      "add_child" = {
+        description = "add a child resource"
+      }
+      "remove_child" = {
+        description = "remove a child resource"
+      }
+      "list_children" = {
+        description = "list child resources"
+      }
     }
     ownerRoleName = "owner"
     roles = {
       owner = {
-        roleActions = ["update_billing_account", "update_metadata", "delete", "link", "share_policy::owner", "share_policy::user", "read_policies"]
+        descendantRoles = {
+          landing-zone = ["owner"]
+        }
+        roleActions = ["update_billing_account", "update_metadata", "delete", "link", "share_policy::owner", "share_policy::user", "read_policies", "add_child", "remove_child", "list_children"]
       }
       user = {
-        roleActions = ["link", "share_policy::user"]
+        descendantRoles = {
+          landing-zone = ["user"]
+        }
+        roleActions = ["link", "share_policy::user", "add_child"]
```

Comment:
I know this has not changed, but it seems weird that users can add other users

Reply:
Hmm I agree, and it goes against the spec: https://docs.google.com/document/d/1AZq0_BQLRx68skieQmdHha8AGY_hcEO2WnKyy8qMmzw/edit#heading=h.tpa5qq630pjp

I can try to convince myself nothing will break then remove this..

Reply:
@ddietterich just checking, does anything on the Verily side break if billing-profile users can't add other users? I don't think anything in Terra CWB relies on this..

Reply:
is this possibly a remnant of how `billing-project`s work? Workspace owner who may only be `billing-project` users can add workspace writers which grants them compute access or something?

Reply:
On second thought I'm going to ticket & look into this separately: https://broadworkbench.atlassian.net/browse/TOAZ-220